profiling-ffi/src/string_storage.rs

```diff
@@ -0,0 +1,204 @@
+use anyhow::Context;
+use datadog_profiling::collections::string_storage::ManagedStringStorage as InternalManagedStringStorage;
+use ddcommon_ffi::{CharSlice, Error, MaybeError, StringWrapper};
+use libc::c_void;
+use std::{ffi::CStr, rc::Rc, sync::RwLock};
+
+#[repr(C)]
+pub struct ManagedStringId {
+    pub value: u32,
+}
+
+#[repr(C)]
+pub struct ManagedStringStorage {
+    // This may be null, but if not it will point to a valid Profile.
```

Comment:
typo ? Profile ? 🤔

Reply:
Fixed in 00a76f2e042e101292725e64f598f884bf1c7621